types/algoliasearch/index.d.ts

```diff
@@ -1356,8 +1375,41 @@ declare namespace algoliasearch {
   }
 
   interface TaskStatus {
-    status: 'published' | 'notPublished',
-    pendingTask: boolean,
+    status: 'published' | 'notPublished';
+    pendingTask: boolean;
+  }
+
+  interface ApiKey {
+    value: string;
+    createdAt: number;
+    acl: (
```

Comment:
acl isn't given to every return right?

Reply:
What do you mean? 
It's returned on GET /keys and /keys/{id} (the REST API doc is really not clear)